version/version.go

```diff
@@ -12,9 +12,10 @@ const (
 )
 
 var (
-	version           = "2.9.0"
-	managedAPIVersion = "1.24.0"
-	log               = l.NewLoggerWithContext(l.Fields{l.ComponentLogContext: "version"})
+	version                      = "2.9.0"
```

Comment:
@carlkyrillos @MStokluska I think we can remove the changes to this file. In general MT RHOAM will use the same operator image as standard rhoam. In the rare occasions where we need to install and emergency fix to MT RHOAM master will already by set to n+1. Also, I think this process can be manual (custom image) given for sandbox

Reply:
Addressed in latest commit.